app/models/concerns/folio/has_site_roles.rb

```diff
@@ -47,4 +102,19 @@ def validate_site_roles
         end
       end
     end
+
+    def validate_site_link_uniqueness
+      counts = site_user_links.group(:site_id).count
```

Comment:
@foton tohle vzdy dela SQL a nerespektuje to neulozene zmeny, nebo ne?

Reply:
Máš recht, opraveno.